lib/ansible/modules/packaging/os/yum.py

```diff
@@ -639,6 +639,42 @@ def local_envra(path):
                                header[rpm.RPMTAG_ARCH])
 
 
+def set_env_proxy(function):
+    def fetching(spec, module, conf_file, installroot):
+        # setting system proxy environment and saving old, if exists
+        yumb = yum_base(conf_file, installroot)
+        namepass = ""
+        schem = "http"
+        old_proxy_env = [os.getenv("http_proxy"), os.getenv("https_proxy")]
+        try:
+            if yumb.conf.proxy:
+                if 'https' in yumb.conf.proxy:
```

Comment:
Should this be startswith instead of in?

Reply:
you right
fix